src/utils/number.ts

```diff
@@ -10,3 +10,16 @@ export function ceiledDivision(dividend: number, divisor: number): number {
   }
   return 0
 }
+
+/**
+ * @param  {number} dividend
+ * @param  {number} divisor
+ * @returns number
+ */
+export function floorDivision(dividend: number, divisor: number): number {
+  if (divisor > 0) {
+    const NumberPages = Math.floor(dividend / divisor) // total number of pages rounded up to the next largest integer.
+    return NumberPages
+  }
+  return 0
+}
```

Comment:
```suggestion
export function floorDivision(dividend: number, divisor: number): number {
  if (divisor > 0) {
    return Math.floor(dividend / divisor)
  }

  return 0
}
```

And move this comment `// total number of pages rounded up to the next largest integer.` to the place where you are calling this function, otherwise, name this function with a specialized name.